attendance/views.py

```diff
@@ -156,7 +154,12 @@ def user(request, username):
         year = datetime.now().year - 1
     else:
         year = datetime.now().year
-
+    
+    if hasattr(u, "profile"):
```

Comment:
I think this logic should be part of `get_hours_quota` method and you

Reply:
It could not. Because user without profile have no get_hours_quota method... But we should create function in utils.py